proposals/stage-0/INITIAL_CPC_VOTING_REPS.md

```diff
@@ -0,0 +1,39 @@
+# Initial CPC Voting Representative Selection
+>  Stage 0
+
+## Champion
+
+Jory Burson (@jorydotcom)
+
+## Description
+
+In issue #152 we have identified 10 representatives from the Impact projects who will be confirmed as voting CPC Representatives. We have also identified 5 nominees for 2 available positions per the [CPC Charter](https://github.com/openjs-foundation/bootstrap/blob/master/CPC-CHARTER.md) from the At Large and Growth Stage projects. Per the charter, the CPC must hold an election. We do not yet have an election system in place.
+
+This document proposes that we accept all 5 of the initial nominees as voting CPC representatives for an interim period of 6 months, by consensus of the Bootstrap team. During this period, the CPC should take the selection and adoption of an election system as a priority. At the end of the term, there shall be a new call for nominees and an election for the 2 available positions as outlined in the charter.
```

Comment:
This implies we have phased terms for impact projects and at large representatives. Is this something that we want, or do we prefer to re-elect the whole CPC every year at the same time?

I'm OK in both sense, but it's something we should discuss / I'm happy to land if this is added to the unresolved questions.

Reply:
That's part of the problem I would love the initial CPC group to address, to determine what's fair. I think it makes sense for the Impact projects to decide this for themselves inasmuch as they are running their own process for determining who their rep is. If Node wants to run an election for their CPC rep every year, and Webpack does it every 2 years, that's fine IMO because it supports individual project autonomy. But what about the nominees for growth + at large rep? 

Great question + should be added to the unresolved section.

Reply:
Do we need a proposal to document the decision in the bootstrap committee? It seems to me like we would benefit from a proposal as to how the future elections would work instead.

To Matteo's point, I was under the impression that the election process for CPC would be annual. We can change this if we want, but I think the term limits are a good thing.

Reply:
I agree that the process was going to be yearly.  If a project reconfirms the same candidate for a subsequent year though that is up to them.